plugins/apiPlugin.ts

```diff
@@ -1,8 +1,8 @@
 import { Plugin } from '@nuxt/types';
 import { AxiosInstance } from 'axios';
+import { AlbumsApi } from '~/api/api/albums-api';
```

Comment:
It's a bit weird that this isn't exported like the others.

It's not blocking, but I wonder if @jellyfin/server should know about this. Maybe there's a difference in how that API is defined?